### PR TITLE
Update queue-pdrs task to use the message adapter

### PR DESCRIPTION
**Pull request by yjpa7145**

### CODE REVIEW

File: cumulus/tasks/queue-pdrs/index.js [hunk 1]
```diff
@@ -1,42 +1,34 @@
...
-* For each PDR, generate a new SF messages send to the step function queue to be executed
-* @param  {object} event lambda event object
-* @param  {object} event.input
-* @param  {array} event.input.pdrs
-* @param  {object} context Lambda context object. See https://docs.aws.amazon.com/lambda/latest/dg/nodejs-prog-model-context.html
-* @param  {lambdaCallback} callback callback function
-* @return {undefined}
-**/
-function handler(event, context, cb) {
+async function queuePdrs(event) {
```
Comment (laurenfrederick) [inline cumulus/tasks/queue-pdrs/index.js]: This function needs a jsdoc

File: cumulus/tasks/queue-pdrs/package.json [hunk 2]
```diff
@@ -38,6 +38,7 @@
   "license": "Apache-2.0",
   "dependencies": {
     "@cumulus/common": "^1.0.1",
+    "@cumulus/cumulus-message-adapter-js": "0.0.1-beta.3",
```
Comment (laurenfrederick) [inline cumulus/tasks/queue-pdrs/package.json]: We should probably have this on 1.0.0 now.

File: cumulus/tasks/queue-pdrs/tests/index.js [hunk 3]
```diff
@@ -1,53 +1,170 @@
...
+  });
+
+  // Make sure we did receive those messages
+  messages.forEach((message) => {
+    const pdrName = message.payload.pdr.name;
+    t.deepEqual(message, expectedMessages[pdrName]);
   });
 });
+
+test.todo('An appropriate error is thrown if the message template could not be fetched');
```
Comment (laurenfrederick) [inline cumulus/tasks/queue-pdrs/tests/index.js]: Is there a ticket for this?

File: packages/ingest/queue.js [hunk 4]
```diff
@@ -20,24 +19,33 @@ async function getMessageFromTemplate(templateUri) {
...
+async function enqueueParsePdrMessage(
+  pdr,
+  queueUrl,
+  parsePdrMessageTemplateUri,
+  provider,
+  collection) {
+  const message = await getMessageFromTemplate(parsePdrMessageTemplateUri);
+
+  message.meta.provider = provider;
+  message.meta.collection = collection;
```
Comment (jennyhliu) [inline packages/ingest/queue.js]: should we add message.meta.pdr = pdr here?
So when a granule is ingested, the granule is inserted into elastic search with pdr name.
https://github.com/cumulus-nasa/cumulus/blob/master/packages/api/es/indexer.js#L389

File: cumulus/tasks/queue-pdrs/package.json (hunk 2 above)
Comment (yjpa7145) [reply cumulus/tasks/queue-pdrs/package.json]: https://bugs.earthdata.nasa.gov/browse/CUMULUS-367

File: cumulus/tasks/queue-pdrs/index.js (hunk 1 above)
Comment (yjpa7145) [reply cumulus/tasks/queue-pdrs/index.js]: Fixed.

File: cumulus/tasks/queue-pdrs/tests/index.js (hunk 3 above)
Comment (yjpa7145) [reply cumulus/tasks/queue-pdrs/tests/index.js]: There is now - https://bugs.earthdata.nasa.gov/browse/CUMULUS-368

File: cumulus/tasks/queue-pdrs/tests/index.js [hunk 5]
```diff
@@ -1,53 +1,170 @@
...
+  const receiveMessageResponse = await sqs().receiveMessage({
+    QueueUrl: t.context.event.config.queueUrl,
+    MaxNumberOfMessages: 10,
+    WaitTimeSeconds: 1
+  }).promise();
+  const messages = receiveMessageResponse.Messages.map((message) => JSON.parse(message.Body));
+
+  t.is(messages.length, 2);
+
+  const receivedPdrnames = messages.map((message) => message.payload.pdr.name);
```
Comment (abarciauskas-bgse) [inline cumulus/tasks/queue-pdrs/tests/index.js]: Nit: (Here and elsewhere in this pr)
> If your function takes a single argument and doesn’t use braces, omit the parentheses

https://github.com/airbnb/javascript#arrows--one-arg-parens

But this is not consistent across the repo, so I don't feel strongly about leaving as is.

File: cumulus/tasks/queue-pdrs/tests/index.js [hunk 6]
```diff
@@ -1,53 +1,170 @@
...
+          path: pdr.path
+        }
+      }
+    };
+  });
+
+  // Make sure we did receive those messages
+  messages.forEach((message) => {
+    const pdrName = message.payload.pdr.name;
+    t.deepEqual(message, expectedMessages[pdrName]);
```
Comment (abarciauskas-bgse) [inline cumulus/tasks/queue-pdrs/tests/index.js]: These tests are 💯

File: packages/ingest/queue.js (hunk 4 above)
Comment (yjpa7145) [reply packages/ingest/queue.js]: I've created https://bugs.earthdata.nasa.gov/browse/CUMULUS-373 for this.

File: cumulus/tasks/queue-pdrs/tests/index.js (hunk 5 above)
Comment (yjpa7145) [reply cumulus/tasks/queue-pdrs/tests/index.js]: Airbnb's guide says to omit the braces, but our project style guide overrides that:

https://github.com/cumulus-nasa/cumulus/blob/master/.eslintrc.json#L95

**Pull request merged**